src/main/scala/abalone/Board.scala

```diff
@@ -61,16 +61,15 @@ object Board {
 
   }
 
-  // TODO Abalone Set
+  // an Hexagon of width 5 fits in a square of width 9
```

Comment:
not sure what you suggested for that one, can you give me an example ?

Reply:
At the moment, all of our boards are regular square/rectangular. So when we say a board is of say Width 9 and Height 2, this means that all of the rows are of width 9.

This will remain true for a hexagonal board of the shape that is needed for the game Hex.

However, this won't be true for Abalone as each rows width is variable. So I was suggesting that we add an extra parameter (a boolean) to indicate this difference in shape. Not sure what to call it, but it would be false for all current boards and true for the Abalone board size.

I can't think of an obvious place where this new parameter must be used, but it could well be useful

Reply:
> if this variable exists then the Board does not have a regular square size
Yes exactly this.
I would make it a parameter of the BoardSize class. You could default it true as an input.
This would probably mean you need to edit the constructor of all BoardSize classes in other game logics (and wrapper layer) but thats fine. As its a default parameter it shouldn't require any other code changes.